run.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
```

Comment:
We can prevent a breaking change by checking whether this script was invoked with any parameters. If so, just pass those parameters through. Something like:

```suggestion

if [ -n "$1" ]; then
    exec /bin/fake-gcs-server "$@"
else
    # the command below
fi
```